_includes/footer-menu.html

```diff
@@ -2,7 +2,7 @@
   <div class="row">
     <div class="col-sm-6 col-sm-offset-3">
       <ul class="footer-secondary-nav nav nav-pills nav-justified">
-        {{ site.data.footer-menu|build }}
+        {{ site.data.main-menu|build }}
```

Comment:
@richmahn - I just took out footer_menu.yml, because the issue said "just like" header so then it becomes redundant if the requirement is for them to be the same.  Will put back in.